both/accounts/config.js

```diff
@@ -1,3 +1,31 @@
+var pwd = AccountsTemplates.removeField('password');
```

Comment:
Also, add a line comment above this line to describe, in plain language, the purpose of the variable.